### PR TITLE
Change installation instruction

### PR DESCRIPTION
**Pull request by FrankKr**
to use package at pypi.

Does this make sense?

### CODE REVIEW

**Review by TonyXiang8787 — APPROVED** [review]

**Comment by bremme** [comment]
Looks fine! Publishing a Python package at PyPi is a very common thing to do, right now it sounds a little bit like its something special: " The repository builds a package available at: https://pypi.org/project/openstf/" You could either remove it completely or rephrase is, for example: "The Python package is published on PyPi and can be installed using pip." 

But this is all not a really big deal I guess.

**Review by bremme — APPROVED** [review]

**Comment by FrankKr** [comment]
> Looks fine! Publishing a Python package at PyPi is a very common thing to do, right now it sounds a little bit like its something special: " The repository builds a package available at: https://pypi.org/project/openstf/" You could either remove it completely or rephrase is, for example: "The Python package is published on PyPi and can be installed using pip."
> 
> But this is all not a really big deal I guess.

I'll rephrase it to make it less big of a deal;)
